svf/lib/Util/SVFBugReport.cpp

```diff
@@ -0,0 +1,357 @@
+//===- SVFBugReport.cpp -- Base class for statistics---------------------------------//
+//
+//                     SVF: Static Value-Flow Analysis
+//
+// Copyright (C) <2013->  <Yulei Sui>
+//
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+//===----------------------------------------------------------------------===//
+
+
+//
+// Created by JoelYang on 2023/4/19.
+//
+
+#include "Util/SVFBugReport.h"
+#ifndef NDEBUG
+#include <cassert>
+#endif
+#include "Util/cJSON.h"
+#include "Util/SVFUtil.h"
+#include <sstream>
+#include <fstream>
+
+using namespace std;
+using namespace SVF;
+
+const std::string GenericBug::getLoc() const
+{
+    return bugEventStack.at(bugEventStack.size() -1)->getEventLoc();
```

Comment:
Add an assertion, otherwise how can you sure the size-1 is the element you want?